frontend/components/Domain/Recipe/RecipePage/RecipePage.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <div>
+    <RecipePageParseDialog
+      :model-value="isParsing"
+      :ingredients="recipe.recipeIngredient"
+      width="70%"
+      @update:model-value="toggleIsParsing"
+      @save="saveParsedIngredients"
+    />
```

Comment:
There's probably be a better way of doing this than just setting the width to 70%